webui/react/src/e2e/models/hew/Dropdown.ts

```diff
@@ -1,22 +1,2 @@
-import { BaseComponent } from 'e2e/models/BaseComponent';
-
-/**
- * Returns a representation of the Mondal component from Hew.
- * This constructor represents the contents in hew/src/kit/Dropdown.tsx.
- * @param {object} obj
- * @param {CanBeParent} obj.parent - The parent used to locate this Dropdown
- * @param {string} obj.selector - Used instead of `defaultSelector`
- */
-export class Dropdown extends BaseComponent {
-  /**
-   * Returns a templated selector for children components.
-   * @param {string} id - menu item id
-   */
-  static selectorTemplate(id: string): string {
-    return `li.ant-dropdown-menu-item[data-menu-id$="${id}"]`;
-  }
-  protected readonly _menu = new BaseComponent({
-    parent: this.root,
-    selector: '.ant-dropdown ul.ant-dropdown-menu:visible',
-  });
-}
+export { Dropdown as DropdownMenu } from 'e2e/models/ant/Dropdown';
+export { Popover as DropdownContent } from 'e2e/models/ant/Popover';
```

Comment:
I find this renaming in particular really confusing. It kinda adds an extra layer to the inheritance chain without actually doing anything.

Reply:
I'm just emulating how the UI does it. When we build page models and see a `import Dropdown, { MenuItem } from 'hew/Dropdown';`, our model should be importing `dropdownMenu` from hew as well.

Reply:
in my opinion, maybe you could argue that the work isn't worth taking the time to do, but now that's it's here (and it only took 20 seconds), there's no good reason to revert.